variables.tf

```
@@ -134,6 +134,12 @@ variable "cluster_kms_key_deletion_window_in_days" {
   description = "The waiting period, specified in number of days (7 - 30). After the waiting period ends, AWS KMS deletes the KMS key"
 }
 
+variable "cluster_kms_additional_admin_arns" {
```

Could we update the variable name to align with the current naming pattern in use:
```suggestion
variable "cluster_kms_key_additional_admin_arns" {
```

---

Sure. I debated about the name a bit because it covers a bit more than just the key, and maybe is more specifically about the key policy, but happy to make the change.

---

I think in this context its specific to the KMS key and follows the `cluster_kms_key_xxx` pattern we have started with `cluster_kms_key_arn` and `cluster_kms_key_deletion_window_in_days`

---

done